### ISSUE TITLE
Moving the mouse doesn't work when running on Wayland

### ISSUE THREAD

**Issue by duracellcell** [juliagoda/antimicro#52]
Is this a known limitation? 

In some XWayland programs I can see an invisible cursor move but it never moves the real one. Clicks and buttons works fine.

**Comment by juliagoda** [comment]
Yes, it's a known limitation. On Wayland this application works partially, but with X11 works fine. As I can see there are many similar problems and not only with mouse pointers:

[XWayland bugs](https://bugs.freedesktop.org/buglist.cgi?component=XWayland&product=Wayland&resolution=---)
[first example](https://bugs.freedesktop.org/show_bug.cgi?id=101970)
[second example](https://bugs.freedesktop.org/show_bug.cgi?id=107358)

[Issue on github](https://github.com/Cloudef/wlc/issues/21}
[Description on wiki site of fedoraproject.org](https://fedoraproject.org/wiki/How_to_debug_Wayland_problems#Known_issues.2C_frequent_complaints.2C_fundamental_changes)

[example of google results](https://www.google.com/search?q=invisible%20cursor%20xwayland)
